home-data-migration/src/main/resources/db/changelog/0.0.1/Issue91.xml

```diff
@@ -0,0 +1,177 @@
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="11"/>
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="12"/>
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="13"/>
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="14"/>
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="15"/>
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="16"/>
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="17"/>
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="18"/>
+        </insert>
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="19"/>
+        </insert>
+    </changeSet>
+
+    <changeSet id="Issue91.005" author="denis-litvinov">
+        <addColumn tableName="houses">
+            <column name="ENABLED" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="Issue91.006" author="denis-litvinov">
+        <createSequence incrementBy="50" sequenceName="cooperations_sequence"/>
+    </changeSet>
+
+    <changeSet id="Issue91.007" author="denis-litvinov">
+        <createSequence incrementBy="50" sequenceName="houses_sequence"/>
+    </changeSet>
+
+    <changeSet id="Issue91.008" author="denis-litvinov">
+        <createSequence incrementBy="50" sequenceName="addresses_sequence"/>
+    </changeSet>
+
+</databaseChangeLog>
+
+
```

Comment:
Maybe it wil be better to use loadData change type and separated csv file with data

Reply:
I think loadData is mostly applicable for test data as we have to set runOnChange to “true” to allow rerunning changeset whenever CSV file is changed. We must not allow to do it.

Reply:
Done.